IWA UI enhancements (#1189)
Esri/arcgis-maps-sdk-dotnet-samples: src/WinUI/ArcGIS.WinUI.Viewer/Samples/Security/IntegratedWindowsAuth/readme.md

1. Enter the URL to your IWA-secured portal.
2. Click the button to search for web maps stored on the portal.

> 3. If you authenticate successfully, portal item results will display in the list.

Reviewer: I'd prefer:

`If authentication is successful portal item results will display in the list.`

Reviewer: That's fine but I'm not sure that we need the comma. Your call.

4. Select a web map item to display it in the map view.

## How it works

1. The currently logged in Windows account will be used to authenticate portal access.
2. An `ArcGISNetworkCredential` object is created.
3. If the user authenticates, the search returns a list of web maps (`ArcGISPortalItem`).

## Relevant API

* ArcGISNetworkCredential
* ArcGISPortal

## About the data

> This sample searches for web map portal items on a secure portal. To successfully run the sample, you need access to a portal secured with Integrated Windows Authentication that contains one or more web map items and credentials which grants you access to that portal.

Reviewer: ```suggestion
This sample searches for web map portal items on a secure portal. Valid credentials and access to an Integrated Windows Authentication secured portal containing one or more web map items are required to run this sample.
```
